src/cli/onefuzz/status/cmd.py

```diff
@@ -20,9 +21,35 @@ def short(value: UUID) -> str:
     return str(value).split("-")[0]
 
 
+class PoolStatus(BaseModel):
+    node_state: Dict[NodeState, int]
+    tasks: Dict[UUID, int]
+
+
 class Status(Command):
     """Monitor status of Onefuzz Instance"""
 
+    def pool(self, *, name: str) -> PoolStatus:
+        pool = self.onefuzz.pools.get(name)
+        nodes = self.onefuzz.nodes.list(pool_name=pool.name)
+
+        tasks = {}
+        node_state = {}
+        for node in nodes:
+            if node.state not in node_state:
+                node_state[node.state] = 0
```

Comment:
These are a good use case for `defaultdict`.

Reply:
Unfortunately, pydantic makes `defaultdict` extremely unergonomic.

Reply:
Ah, good to know.